config.json

```diff
@@ -154,6 +154,18 @@
       ]
     },
     {
+      "uuid": "357d63b9-002e-2680-561c-8aa540f6a4ea7181b64",
```

Comment:
this is too long to be a UUID, assuming that UUID is https://en.wikipedia.org/wiki/Universally_unique_identifier. The last segment should be 12 characters, not 19 as it currently is.

Reply:
I generated this uuid with the ```configlet uuid``` command. But this numbers are always that long.
Is this the right comand?

Reply:
then `configlet uuid` is generating wrong UUIDs.

Reply:
therefore, by definition, it is not the right command

Reply:
and that warrants an issue for configlet, since I presume `configlet uuid` is advertised to generate UUIDs (and thus is failing its advertisement)